packages/amplify-graphql-conversation-transformer/src/resolvers/verify-session-owner-resolver-fn.js

```diff
@@ -0,0 +1,30 @@
+export function request(ctx) {
+  const { authFilter } = ctx.stash;
+
```

Comment:
Here and throughout the PR: let's add a defensive existence assertion for `authFilter`. I'm comfortable that the existing implementation sets it appropriately, but if a future update accidentally forgets it, I'd prefer to catch that early instead of performing a query without the filter in place. (It's possible that the `toDynamoDBFilterExpression` util below would fail in that case, but I'd prefer to make the check explicit.)

Reply:
Unless you strongly object, I'll include this in a follow up (I have a draft PR queued up that makes some changes here). I know that this is a safe (and correct) change to make, but I'd really prefer to keep this PR non-functional.

Reply:
Noted in the aforementioned draft PR as a reminder.
https://github.com/aws-amplify/amplify-category-api/pull/2910#issuecomment-2377392782